### PR TITLE
release: 2023-12-18

### PR DESCRIPTION
**Pull request by kprotty**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,47 @@
 # TigerBeetle Changelog
 
+## 2023-12-18
+
+### Internals
+
+- [#1359](https://github.com/tigerbeetle/tigerbeetle/pull/1359)
+
+  We've established a rotation between the team for handling releases. As the one writing these
+  release notes, I am now quite aware.
```
Comment (matklad) [inline CHANGELOG.md:12]: 🤣

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,47 @@
...
+
+  Clients automatically batch pending similar requests 🎉! If a tigerbeetle client submits a
+  request, and one with the same operation is currently in-flight, they will be grouped and
+  processed together where possible (currently, only for `CreateAccount` and `CreateTransfers`).
+  This should [greatly improve the performance](https://github.com/tigerbeetle/tigerbeetle/pull/1257#issuecomment-1812648270)
+  of workloads which submit a single operation at a time.
+
+### TigerTracks 🎧
+
+- [Remaster](https://open.spotify.com/track/0YZKbKo9i91i7LD0m1KASq)
```
Comment (matklad) [inline CHANGELOG.md]:
```suggestion
- [Carouselambra](https://open.spotify.com/track/0YZKbKo9i91i7LD0m1KASq)
```

Comment (kprotty) [reply CHANGELOG.md]: Any way to get passed spellcheck for this one?

File: CHANGELOG.md [hunk 3]
```diff
@@ -40,7 +42,7 @@
 
 ### TigerTracks 🎧
 
-- [Carouselambra](https://open.spotify.com/track/0YZKbKo9i91i7LD0m1KASq)
+- ["Carouselambra"](https://open.spotify.com/track/0YZKbKo9i91i7LD0m1KASq)
```
Comment (sentientwaffle) [inline CHANGELOG.md]: You can add the word to `scripts/.cspell.json`.

**Review by sentientwaffle — APPROVED** [review]

**Pull request merged**